blocks/inspector-controls/toggle-control/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { withInstanceId, Toggle } from 'components';
```

Comment:
Oof, while this is preferred, we're also not currently exporting `Toggle` from `components/index.js` as we should be.